### PR TITLE
Update notebooks

### PR DESCRIPTION
**Pull request by mrocklin**

### CODE REVIEW

File: dask_ec2/formulas/salt/dask/distributed/init.sls [hunk 1]
```diff
@@ -66,6 +66,20 @@ update-pandas:
...
+    - require:
+      - sls: conda
+
+update-dask-glm:
+  pip.installed:
+    - name: dask-glm
+    - bin_env: {{ install_prefix }}/bin/pip
+    - require:
+      - sls: conda
+
```
Comment (mrocklin) [inline dask_ec2/formulas/salt/dask/distributed/init.sls]: @quasiben is this the right way to install some additional libraries from PyPI?

Comment (quasiben) [reply dask_ec2/formulas/salt/dask/distributed/init.sls]: yeah, that looks right

**Comment by mrocklin** [comment]
Does anyone know why this PR might cause this failure?

```python
_____________________________ test_provision_salt ______________________________
cluster = {'region': 'foo', 'instances': [{'port': 2222, 'uid': 1, 'keypair': '~/.vagrant.d/insecure_private_key', 'username': '...}, {'port': 2223, 'uid': 1, 'keypair': '~/.vagrant.d/insecure_private_key', 'username': 'root', 'ip': 'ci_compute_1'}]}
    @remotetest
    def test_provision_salt(cluster):
        result = invoke("provision")
        if result.exit_code != 0:
            print(result.output_bytes)
>       assert result.exit_code == 0
E       assert 1 == 0
E        +  where 1 = <Result SystemExit(1,)>.exit_code
```

**Comment by danielfrg** [comment]
I dont think thats the reason

**Comment by mrocklin** [comment]
If so then is it safe to merge this PR?  (I am prepping for a talk for tonight)

**Comment by danielfrg** [comment]
Actually it is:

```
Failed to install packages: dask-glm. Error: Collecting dask-glm   Could not find a version that satisfies the requirement dask-glm (from versions: )
```

pip cannot find the package?

**Comment by danielfrg** [comment]
Yeah, i dont see `dask-glm` it on pypi

**Comment by mrocklin** [comment]
Ah, ok.  I guess I need to learn to read logs on dask-ec2 test failures.

**Comment by mrocklin** [comment]
Thank you @danielfrg

**Comment by mrocklin** [comment]
Merging soon if there are no objections

**Comment by danielfrg** [comment]
👍

**Pull request merged**